jenkins/ci.suse.de/cloud-trackupstream.yaml

```diff
@@ -81,12 +84,22 @@
             ].contains(component)
           )
           ||
+          (
+            [
+              "Devel:Cloud:8:Staging"
```

Comment:
What does this mean?

Reply:
It's supposed to exclude the package from Devel:Cloud:8. Basically it's only tracked for Cloud 7

Reply:
See: https://ci.suse.de/job/cloud-trackupstream/1466/ for a run of the updated job

Reply:
I see, thanks